### ISSUE TITLE
Ensure that uv works with `conda` Python distributions

### ISSUE THREAD

**Issue by charliermarsh** [astral-sh/uv#1062]
This probably already works, but I've never tested it. It's worth testing manually.

**Comment by drewbitt** [comment]
Works for me with miniforge installed through brew
`Using Python 3.12.1 interpreter at /opt/homebrew/Caskroom/miniforge/base/bin/python3.12`

**Comment by charliermarsh** [comment]
You're awesome, thank you.

**Comment by shughes-uk** [comment]
This might be for a seperate ticket but we want into this

```
uv pip install chardet                                                    (coiled)  ✘ 2
error: Failed to list installed packages
  Caused by: Found duplicate package in environment: wheel (/opt/homebrew/Caskroom/mambaforge/base/envs/coiled/lib/python3.11/site-packages/wheel-0.42.0.dist-info vs. /opt/homebrew/Caskroom/mambaforge/base/envs/coiled/lib/python3.11/site-packages/wheel-0.40.0.dist-info)
```

Duplicate dist-info's are pretty common in our experience with conda.

**Comment by charliermarsh** [comment]
That's interesting... We can probably support it, though would need to map out how various operations work. What does `pip uninstall wheel` do in that case?

**Comment by shughes-uk** [comment]
It seems to just remove the `dist-info` of whatever version pip installed.

Generally this seems to happens when you install a package via pip/conda and then install it again with the other package manager, files don't get cleaned up properly.

**Comment by ntessore** [comment]
Not sure if I'm ignorant about a fundamental difference between build isolation in `pip` and `uv pip`, but simply trying out `uv pip install .` on a local project fails where `pip install .` succeeds. That happens in a conda environment where all dependencies are installed, but `uv pip` tries to download and build a package that cannot be compiled on my machine (but is installed in the conda env).

**Comment by zanieb** [comment]
@ntessore fwiw I think that should be resolved by #2596
